libs/sdk-ui-pivot/src/CorePivotTable.tsx

```diff
@@ -184,6 +184,11 @@ export class CorePivotTablePure extends React.Component<ICorePivotTableProps, IC
     private visibleData: DataViewFacade = null;
     private currentFingerprint: string = null;
```

Comment:
I am not sure we need the currentFingerprint anymore. It changes onPageLoaded, but tbh I do not think it will ever be different from lastInitRequestFingerprint.

Reply:
it can be.. because of totals & sort changes which may be initiated through the menu / clicking around